static/app/utils/profiling/profile/continuousProfile.tsx

```diff
@@ -48,7 +48,7 @@ export class ContinuousProfile extends Profile {
       const stack = chunk.stacks[sample.stack_id];
       let size = 0;
 
-      for (let j = 0; j < stack.length; j++) {
+      for (let j = stack.length - 1; j >= 0; j--) {
```

Comment:
This was what we talked about that caused the stack to be upside down right?

Reply:
Yup, I reversed it